sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_path_client.py

```diff
@@ -204,6 +218,28 @@ def _create(self, resource_type, content_settings=None, metadata=None, **kwargs)
             For example, if p is 0777 and u is 0057, then the resulting permission is 0720.
             The default permission is 0777 for a directory and 0666 for a file. The default umask is 0027.
             The umask must be specified in 4-digit octal notation (e.g. 0766).
+        :keyword str owner:
+            The owner of the blob or directory. Default value is None.
```

Comment:
Two comments on this line:
- _file_ or directory? Instead of blob. Applies to `group` as well.
- Let's remove the `Default value is None.` for **all** of these. These are keywords so that's a given. I don't love how the generated code added this to the docstring.